module-4/remove_metadata/remove_metadata.py

```diff
@@ -0,0 +1,46 @@
+import os
+import subprocess
+import argparse
+
+def remove_metadata_exiftool(input_file, output_file):
+    """
+    Removes metadata using ExifTool.
+    """
+    try:
+        command = f'exiftool -all= -o {output_file} {input_file}'
+        subprocess.run(command, shell=True, check=True)
+        print(f"Metadata successfully removed using ExifTool. Output: {output_file}")
+    except subprocess.CalledProcessError as e:
+        print(f"Error removing metadata with ExifTool: {e}")
```

Comment:
this section and the corresponding equivalent in the function below are the same code and can be abstracted into a single function that receives the command and maybe tool as parameters.